.gitignore

```diff
@@ -7,3 +7,4 @@ dist/
 mf2py.egg-info/
 nbproject/
 venv/
+*~
```

Comment:
fyi, generally recommended to ignore these sorts of files globally. Like I have them in ~/.gitignore_global and then in ~/.gitconfig:
```
[core]
	excludesfile = /home/kmahan/.gitignore_global
```